understanding/20/pause-stop-hide.html

```diff
@@ -1,107 +1,85 @@
 <!DOCTYPE html>
 <html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
-   <meta charset="UTF-8"></meta>
+   <meta charset="UTF-8"/>
    <title>Understanding Pause, Stop, Hide</title>
    <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
 </head>
 <body>
    <h1>Understanding Pause, Stop, Hide</h1>
-   
+
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
          <dt>Goal</dt><dd>Fewer users are distracted by content that updates or moves.</dd>
          <dt>What to do</dt><dd>Let users control content changes that occur in parallel with other content.</dd>
-         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.</dd> 
+         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.</dd>
       </dl>
 
    </section>
-   
+
    <section id="intent">
       <h2>Intent of Pause, Stop, Hide</h2>
-      
-      
+
       <p>The intent of this Success Criterion is to avoid distracting users during their interaction
          with a Web page.
       </p>
-      
+
+      <p>In the context of this Success Criterion, "starts automatically" broadly refers to animations/updates
+         that are not the direct result of a user's explicit activation.</p>
```

Comment:
> "not the direct result of a user's explicit activation"

This does not clarify whether animation starting on hover or focus would count as explicit.

Reply:
```suggestion
         that are not the direct result of a user's intentional activation, e.g., selecting a link or button.</p>
```